src/pages/User/components/UserHeader/UserHeader.story.js

```diff
@@ -0,0 +1,16 @@
+export default {
+  title: 'Pages/User/Components/Header',
+  component: UserHeader,
+  argsTypes: {},
+};
+
+const UserHeaderTemplate = (args) => ({
+  components: { UserHeader },
+  setup() {
+    return { args };
+  },
+  template: '<user-header v-bind="args" />',
+});
+
+export const Default = UserHeaderTemplate.bind({});
+Default.args = {};
```

Comment:
Please import here the component that you're using in the storybook, at the line 8 you're using UserHeader but I think that component should be imported